mobilecoind/api/proto/mobilecoind_api.proto

```diff
@@ -187,6 +187,21 @@ message MonitorStatus {
     string name = 6;
 }
 
+// Enum used to indicate whetehr a ProcessedTxOut is a sent one or a received one.
+enum ProcessedTxOutDirection {
+    // This should never happen, but is available here as an option to catch uninitialized data.
+    // The name "Unknown" cannot be used because, quoting the protobuf compiler:
+    // Note that enum values use C++ scoping rules, meaning that enum values are siblings of their type, not children of it.  Therefore, "Unknown" must be unique within "mobilecoind_api", not just within "ProcessedTxOutDirection".
```

Comment:
does this suggest that we won't be able to use `Invalid` later? Maybe we should add extra scope to unknown when we use it, like `{EnumType}_Unknown`?

Reply:
Yup. I can scope it.

Reply:
However, this applies to all other variants in the enum. I suggest we punt on it until there's an actual conflict. It won't be a breaking API change since the tag numbers remain the same.